src/Message.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="sc-message" :id="message.id">
+  <div class="sc-message" :id="message.type === 'file' ? message.data.file.id : message.id">
```

Comment:
why is this edit necessary?

Reply:
here we can take the difference between a message id and a file id. This mark can be used as help for the user when download the file for the backend